2.1.X version breaking changes in minor update

Prettier was updated to 2.0 and it's introduced a lot of breaking changes. So many users will still use prettier 1.9.X.

However this plugin has minor upgrade, so it may automatically upgraded by users with `npm up` and they will receive an error, cause prettier will have 1.9.X. If prettier-plugin-packagejson will have version 2.1.X and prettier will be at 1.9.X users will receive following error:
```
Cannot find module 'prettier/parser-babel'
```

Can you consider bumping major version of this plugin with updated prettier and update 2.1.1 with prettier 1.9.X?

--- a/lib/index.js
+++ b/lib/index.js
@@ -1,4 +1,5 @@
-const { parsers } = require('prettier/parser-babel')
+const requireSafe = require('./require')
+const { parsers } = requireSafe('prettier/parser-babel') || requireSafe('prettier/parser-babylon')
 const sortPackageJson = require('sort-package-json')
 const parser = parsers['json-stringify']
 

--- a/lib/require.js
+++ b/lib/require.js
@@ -1,0 +1,5 @@
+module.exports = path => {
+  try {
+    return require(path)
+  } catch (error) {}
+}

--- a/package.json
+++ b/package.json
@@ -39,7 +39,7 @@
     "@types/jest": "25.1.4",
     "jest": "25.1.0",
     "npm-run-all": "4.1.5",
-    "prettier": "2.0.1"
+    "prettier": "2.0.2"
   },
   "peerDependencies": {
     "prettier": ">= 1.16.0"

